test/index.test.ts

```diff
@@ -874,6 +874,8 @@ describe('Configure AWS Credentials', () => {
 
   test('prints warning for access key usage and no session token', async () => {
     jest.spyOn(core, 'getInput').mockImplementation(mockGetInput(ASSUME_ROLE_INPUTS));
+    process.env['AWS_ACCESS_KEY_ID'] = FAKE_ACCESS_KEY_ID;
```

Comment:
We should also have another test without these lines you've added to verify that the warning does NOT throw

Reply:
Thank you for your review! I added test check doesn't show warning when using STS access key.
https://github.com/aws-actions/configure-aws-credentials/pull/904/commits/3e570a92af761a2286627d2dd67045a0288ef440